app/experimenter/legacy-ui/rapid/components/visualization/ResultsRow.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import ReactTooltip from "react-tooltip";
+
+import {
+  Significance,
+  Metric,
+} from "experimenter-rapid/components/visualization/constants/analysis";
+import { SIGNIFICANCE_TIPS } from "experimenter-rapid/components/visualization/constants/tooltips";
+
+const showSignificanceField = (significance, interval) => {
+  {
+    let significanceIcon, className;
+    switch (significance) {
+      case Significance.POSITIVE:
+        significanceIcon = (
+          <i
+            className="fas fa-arrow-up"
+            data-tip={SIGNIFICANCE_TIPS.POSITIVE}
+          />
+        );
+        className = "positive-significance";
+        break;
+      case Significance.NEGATIVE:
+        significanceIcon = (
+          <i
+            className="fas fa-arrow-down"
+            data-tip={SIGNIFICANCE_TIPS.NEGATIVE}
+          />
+        );
+        className = "negative-significance";
+        break;
+      case Significance.NEUTRAL:
+        significanceIcon = (
+          <i className="fas fa-minus" data-tip={SIGNIFICANCE_TIPS.NEUTRAL} />
+        );
+        className = "neutral-significance";
+        break;
+    }
+
+    return (
+      <div className={className} data-testid={className}>
+        <ReactTooltip />
+        {significanceIcon}
+        &nbsp;
+        {interval}
+      </div>
+    );
+  }
+};
+
+const populationField = (point, percent) => {
+  return (
+    <div>
+      <div className="font-weight-bold">{point}</div>
+      <div className="h6">{percent}%</div>
```

Comment:
Not a blocker, but these might semantically be better off as `<p>` tags or even a `<span>`